recirq/kpz/kpz.ipynb

```diff
@@ -0,0 +1,746 @@
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9d29cea2",
+   "metadata": {},
+   "source": [
+    "# Kardar-Parisi-Zhang Dynamics in the Heisenberg Spin Chain"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c4a1ff2c",
+   "metadata": {},
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://quantumai.google/cirq/experiments/kpz/kpz\"><img src=\"https://quantumai.google/site-assets/images/buttons/quantumai_logo_1x.png\" />View on QuantumAI</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/quantumlib/ReCirq/blob/master/docs/kpz/kpz.ipynb\"><img src=\"https://quantumai.google/site-assets/images/buttons/colab_logo_1x.png\" />Run in Google Colab</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/quantumlib/ReCirq/blob/master/docs/kpz/kpz.ipynb\"><img src=\"https://quantumai.google/site-assets/images/buttons/github_logo_1x.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/ReCirq/docs/kpz/kpz.ipynb\"><img src=\"https://quantumai.google/site-assets/images/buttons/download_icon_1x.png\" />Download notebook</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1de2dd3b",
+   "metadata": {},
+   "source": [
+    "Before beginning, we will import the necessary modules into the notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 58,
+   "id": "cef7a354",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import recirq\n",
+    "except ImportError:\n",
+    "    !pip install --quiet git+https://github.com/quantumlib/ReCirq\n",
+    "    import recirq\n",
+    "\n",
+    "try:\n",
+    "    import qsimcirq\n",
+    "except ImportError:\n",
+    "    !pip install qsimcirq --quiet\n",
+    "    import qsimcirq\n",
+    "\n",
+    "import cirq\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib import ticker\n",
+    "import numpy as np\n",
+    "\n",
+    "import recirq.kpz.experiment as kpz\n",
+    "\n",
+    "plt.rcParams['figure.dpi'] = 144"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f4110f55",
+   "metadata": {},
+   "source": [
+    "### Motivation and Background\n",
+    "\n",
+    "For background and motivation, please either read the paper, [arXiv:2306.09333](https://arxiv.org/abs/2306.09333), or watch the [talk](https://www.youtube.com/watch?v=U12xdMA3v1E).\n",
```

Comment:
Probably would add at least a short intro here.  And then people can read the paper for follow-up.